apps/minifront/src/components/shared/restrict-max-width.tsx

```diff
@@ -1,5 +1,5 @@
 import { ReactNode } from 'react';
 
 export const RestrictMaxWidth = ({ children }: { children: ReactNode }) => {
-  return <div className='mx-auto xl:max-w-[1276px]'>{children}</div>;
+  return <div className='mx-auto w-full max-w-full xl:max-w-[1276px]'>{children}</div>;
```

Comment:
This fixes an issue where the `<RestrictMaxWidth />` div didn't always grow to be as big as it could be, unless its contents stretched it to its max width.